### PR TITLE
use ci/deploy.sh in sync with upstream

### PR DESCRIPTION
**Pull request by minrk**
should make it harder to get out of sync

### CODE REVIEW

**Comment by Carreau** [comment]
We are going to converge !

`./ci/deploy.sh: line 4: encrypted_c6b45058ffe8_key: unbound variable`

File: .travis.yml [hunk 1]
```diff
@@ -28,12 +28,7 @@ jobs:
     if: type = push AND branch = master
     # set env to avoid deploy matrix
     env: TEST=helm
-    script:
-      - openssl aes-256-cbc -K $encrypted_d8355cc3d845_key -iv $encrypted_d8355cc3d845_iv -in travis.enc -out travis -d
```
Comment (minrk) [inline .travis.yml]: I copied ci/deploy.sh from zero-to-jupyterhub, but forgot to replace the values with the ones here for this repo.

Comment (Carreau) [reply .travis.yml]: Maybe a sign we should pass the variable as parameters if we want to keep things in sync.

Comment (willingc) [reply .travis.yml]: +1 @Carreau

**Pull request merged**